python/src/FunctionalChaosResult_doc.i.in

```diff
@@ -160,3 +160,51 @@ Returns
 -------
 transformation : :class:`~openturns.Function`
     Transformation :math:`T` such that :math:`T(\vect{X}) = \vect{Z}`."
+    
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::FunctionalChaosResult::drawSelectionHistory
+"Draw the basis selection history.
+
+This is only availble with LARS, and when the output dimension is 1.
+
+Returns
+-------
+graph : :class:`~openturns.Graph`
+    The evolution of the basis coefficients at each selection iteration"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::FunctionalChaosResult::getCoefficientsHistory
+"The coefficients values selection history accessor.
+
+This is only availble with LARS, and when the output dimension is 1.
+
+Returns
+-------
+coefficientsHistory : 2-d sequence of float
+    The coefficients values selection history"
```

Comment:
It seems to me that the number of rows and columns should be presented. A pointer to the theory help page `polynomial_sparse_least_squares` will make this easier to understand.